sci-misc/kaldi/kaldi-0_p20150101.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI="5"
+
+inherit eutils flag-o-matic subversion
+
+DESCRIPTION="A toolkit for speech recognition"
+HOMEPAGE="http://kaldi.sourceforge.net/"
+ESVN_REPO_URI="https://svn.code.sf.net/p/kaldi/code/trunk/src@4735"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+IUSE="doc test threads"
+KEYWORDS="~amd64"
+
+RDEPEND="
+	media-libs/speex
+	sci-libs/atlas[lapack,threads=]
+	~sci-misc/openfst-1.3.4"
+DEPEND="${RDEPEND}
+	doc? ( app-doc/doxygen )"
+
+src_prepare() {
+	epatch \
+		"${FILESDIR}"/configure.patch \
+		"${FILESDIR}"/Makefile.patch \
+		"${FILESDIR}"/default_rules.mk.patch
+}
+
+src_configure() {
+	./configure \
```

Comment:
I tried, but it didn't work because econf requires some options which this configure script don't support.